### DISCUSSION TITLE
How to add received stock?

### DISCUSSION THREAD

**Discussion by stelgenhof** [ghostfolio/ghostfolio#2882] [Q&A] [answered]
From a previous employer, I received stock options which were converted into stock. What is the best way to add these to Ghostfolio? There isn't really an activity that would match this scenario.

- Buy: This seems to be the most fitting but technically I didn't buy these stocks. Not sure, if using this activity, if any of the reporting would be correct
- Valuable: This option does allow to add the total value of the stocks, but not allow for any further trading/reporting.

**Comment by dtslvr** [comment, marked as answer]
I have entered my gifted stocks from a broker as a buy activity with the price at that time and marked the activity with a _Grant_ tag (for filtering). Technically, this is not 100% correct, but at least these stocks count to my net worth.

**Reply by stelgenhof** [reply]
Thanks! Since this stock has poorly performed since they were gifted to me, the performance (Gross/Net) numbers on the summary page are skewed. The net worth indeed is correct, but the performance % numbers are negative; this stock overshadowing the good performance of my other holdings.

**Reply by dtslvr** [reply]
You may like to activate _Exclude from Analysis_ at account level.

**Reply by stelgenhof** [reply]
Yes, that is what I did in the end. Thanks!
